website/app/components/doc/a11y-support.hbs

```diff
@@ -0,0 +1,3 @@
+<h2 class="doc-markdown-h2 doc-page-sidecar-scroll-margin-top" id="support">Support</h2>
```

Comment:
The `doc-markdown` class names are reserved for the automatic conversion of markdown to HTML. In all the other cases we should use the [dedicated CSS helpers](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Doc-styling-and-components.md#typography) for typography, lists, links, etc:
```suggestion
<h2 class="doc-text-h2 doc-page-sidecar-scroll-margin-top" id="support">Support</h2>
```